src/components/LoginForm/LoginForm.js

```diff
@@ -82,9 +82,9 @@ function renderError(error) {
 function LoginForm() {
   const dispatch = useDispatch();
   const [rooms, setRooms] = useState([]);
+  const [selectedRoom, setSelectedRoom] = useState({});
```

Comment:
@imobachgs, as you worked around here recently, I need your insights here, please.

The thing is that I've changed a bit the layout to share the same _row_ for the PIN field and the submit action. It can be argued that this only complicates the markup... and it's true. However, the motivation behind such decision is to keep the form fully visible when using a small device in the landscape orientation. I mean, 


| Before | After |
|-|-|
| ![Screenshot_2020-04-04 Jangouts(1)](https://user-images.githubusercontent.com/1691872/78720599-e61e9d80-791d-11ea-9b3d-7530ee2cd334.png) | ![Screenshot_2020-04-05 Jangouts(8)](https://user-images.githubusercontent.com/1691872/78720632-f20a5f80-791d-11ea-8390-2f2f727a477e.png) |
| ![Screenshot_2020-04-05 Jangouts(1)](https://user-images.githubusercontent.com/1691872/78720674-051d2f80-791e-11ea-9701-afa2bc6b45ed.png) | ![Screenshot_2020-04-07 Jangouts](https://user-images.githubusercontent.com/1691872/78720778-31d14700-791e-11ea-8538-c9894c986971.png) |

Just trying to follow one of the main core concepts of tailwind: [mobile first](https://tailwindcss.com/docs/responsive-design#mobile-first).

Thus, this commit (1ed597369ab2d3c40a70a56624f30fe0f1475fde) rid of the `watch` and use a hook to keep the selected room in the internal state. Do you see think this is right?

Thanks!